src/frontends/tensorflow_common/src/op/pad.cpp

```diff
@@ -44,7 +45,7 @@ OutputVector translate_pad_op(const NodeContext& node) {
     default_op_checks(node, 2, {"Pad"});
     auto input = node.get_input(0);
     auto paddings = node.get_input(1);
-    auto constant_value = make_shared<Constant>(input.get_element_type(), Shape{}, 0);
+    auto constant_value = create_same_type_const_scalar<int32_t>(input, 0);
```

Comment:
Why it named as "constant_value" instead of "const_zero" as in other places?

Reply:
It makes sense to have the common names for such constant. Let us do it in the separate PR. It is a sort of improvement of our code and not blocker.